pkgs/tools/security/ghidra/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, unzip, lib, makeWrapper, autoPatchelfHook
+, openjdk11, pam
+}: let
+
+  pkg_path = "$out/lib/ghidra";
+
+in stdenv.mkDerivation {
+
+  name = "ghidra-9.0";
+
+  src = fetchurl {
+    url = https://ghidra-sre.org/ghidra_9.0_PUBLIC_20190228.zip;
```

Comment:
```ghidra-sre.org``` is geo-restricted, for example it answers 403 to Russian IPs
If there is a more accessible mirror (maybe on github?), it would be nice to change the url

Reply:
I guess once it's in the binary cache it wont be a problem for end users unless they change something. Does fetchurl take some kind of mirrors field?

Reply:
Yes. See `pkgs/build-support/fetchurl/mirrors.nix`.

Reply:
also https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchurl/default.nix#L43 ?